src/qt/test/wallettests.cpp

```diff
@@ -212,7 +212,7 @@ void TestGUI(interfaces::Node& node)
     // Check Request Payment button
     ReceiveCoinsDialog receiveCoinsDialog(platformStyle.get());
     receiveCoinsDialog.setModel(&walletModel);
-    RecentRequestsTableModel* requestTableModel = walletModel.getRecentRequestsTableModel();
+    auto requestTableModel = MakeUnique<RecentRequestsTableModel>(&walletModel);
```

Comment:
Obviously this is wrong, neeed to get receiveCoinsDialog.m_recent_requests_table_model.. will fix.